packages/docs/docs/api/fhir/resources/account.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Account
+sidebar_position: 0
+---
+
+import definition from '@site/static/data/resourceDefinitions/account.json';
+import { ResourcePropertiesTable, SearchParamsTable } from '@site/src/components/resourceTables';
+
+# Account
+
+<p>{definition.description}</p>
+
+## Properties
+
+<ResourcePropertiesTable properties={definition.properties.filter((p) => !(p.inherited && p.base.includes('Resource')))} />
```

Comment:
Rather than doing the filter on every page, you could do it inside `<ResourcePropertiesTable>`

Reply:
I guess you would need a boolean `inherited` prop to support both use cases.

Reply:
Yeah, I thought this was more flexible